pipeline/backend/kubernetes/service_test.go

```diff
@@ -32,14 +32,17 @@ func TestService(t *testing.T) {
 	  "spec": {
 	    "ports": [
 	      {
+				  "name": "port-1",
```

Comment:
Could you please align the indentation with the other items? I.e. using tab+whitespace in the same combination.

Reply:
It should be fixed